qa/tasks/cephfs/filesystem.py

```diff
@@ -465,6 +467,17 @@ def create(self):
 
         self.getinfo(refresh = True)
 
+        
+    def check_pool_application(self, pool_name):
+        osd_map = self.mon_manager.get_osd_dump_json()
+        for pool in osd_map['pools']:
+            if pool['pool_name'] == pool_name:
+                if "application_metadata" in pool:
+                    if not "cephfs" in pool['application_metadata']:
+                        raise RuntimeError("Pool %p does not name cephfs as application!".\
```

Comment:
Does this need to be guarded against rados test suites where the luminous "min osd release" upgrade is performed at the end of the test (after fs new)?